### ISSUE TITLE
Add-ons with localised names

### ISSUE THREAD

**Issue by revad** [cleidigh/ThunderKdB#27]
This is a  enhancement suggestion - and a couple of questions.

I've been lurking in Geckozone, the French forum, for a while to see if anyone had observations about my addons. Someone did, so I created an account  - and as a result updated one of them.  Having an account I thought I'd post a link to the TB 78 compatible addons:
https://forums.mozfr.org/viewtopic.php?f=4&t=145286

I also noted in that post that my two addons have different names in French~. Which made me wonder:
- are there other addons that have different names in other locales, so someone looking for the local name wouldn't find it in that list?
- if so, could there be a localised list - a parameter, or even detect the browser language? Maybe there are so few it's not worth the effort.
- with the possible exception of Grammalecte I see no addons that have foreign names. Are there none? I thought - but I don't remember why I thought - that there were some Japanese-only addons, or used to be. Does the list include non-EN addons?

~ For info, I gave Limit non-BCC recipients different names in other locales because 'BCC' is not used everywhere: the BCC field in the compose window has local variants: "Copie cachée à" in French, for example. It does cause me a lot more work, though!

**Comment by cleidigh** [comment]
@revad 
thanks for the post/suggestions! I like to see people using the tool.

I think I follow your comments but if my responses below are off let me know

- as you know you can use the localized name/description tokens in the manifest
  therefore every locale can have a different name
- for simplicity the extension lists first use the   default  locale name
  if there is no default locale, en is used for if available
  otherwise the first listed locale ( somewhat arbitrary)

so that's how it works now
if you look atcomplete extension list there are several non- English names

basically I would have to create a new list for perhaps focusing on
supported locales including other locale names
that's something I have contemplated before but not had the time to do
at the moment I am going crazy doing 78 conversions, so I don't think I will be able
put together this anytime soon.
I will try to figure out if there is any simple solution.
does that answer your questions?
Christopher

**Comment by revad** [comment]
So people on the French forum I referred to, who would have French as their default locale, would get add-on names in French, if they differed from English. That's fine. I assumed the list was in English. I didn't envisage French people wanting a list with German names. 
So I'll close this.

(I did find some addons with Japanese-only names, but they very old - TB 2 or 3.)
Dave

**Comment by cleidigh** [comment]
@revad 
I think I might not have been clear:
- the extension list have no awareness of the browsers locale
-  when I referred to a default locale I am referring to what the extension tells ATN 
in order to show other names I have to create a new list looking at the other supported locales
